source/_integrations/verisure.markdown

```diff
@@ -85,9 +85,11 @@ automation:
 
 ## Limitations and known issues
 
-Some users have reported that this integration currently doesn't work in the following countries:
+Some users have reported that this integration currently doesn't work in the following countries because of two-factor authentication:
 
 - France
 - Ireland
 - Italy 
 - Sweden
+
+Instead, users in these countries can use the [`securitas-direct-new-api`](https://github.com/guerrerotook/securitas-direct-new-api) available via HACS.
```

Comment:
We don't support custom integrations and, therefore, we will not be able to recommend or mention them in our official documentation.

Nevertheless, thanks for being willing to contribute 👍